### PR TITLE
Sync exercise docs and metadata

### PR DESCRIPTION
**Pull request by BNAndras**

### CODE REVIEW

File: exercises/practice/etl/.meta/config.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
-  "authors": [],
+  "authors": [
+    "sturmer"
```
Comment (ErikSchierboom) [inline exercises/practice/etl/.meta/config.json:3]: This seems to be an unrelated change. I don't mind it _that_ much, but you could consider doing this in a separate PR.

**Review by ErikSchierboom — APPROVED** [review]

File: exercises/practice/etl/.meta/config.json (hunk 1 above)
Comment (BNAndras) [reply exercises/practice/etl/.meta/config.json:3]: Will do. Definitely a case of “well one more tweak can’t hurt”.

**Review by ErikSchierboom — APPROVED** [review]
